### PR TITLE
Update to v1, sort communities by size and desired order

### PR DESCRIPTION
**Pull request by bhousel**
I released [v1.0.0](https://github.com/osmlab/osm-community-index/releases/tag/v1.0.0) of the community index today!  🎊 

This PR contains the changes to
- Fetch the current version (1.x) from jsdelivr CDN
- The format of `combined.geojson` has changed slightly, so this code handles the updated version that now includes the `area` property.
- Sort communities by `area` ascending, `order` descending (like what iD does)
    
 In other words:
- small areas first, large areas last
- then uses the `order` property (if set) to control the sort order within a single area

### CODE REVIEW

**Comment by bhousel** [comment]
code is moved to https://github.com/osmlab/osm-community-index/ so no need for this now!
